pkg/containerprofilemanager/v1/container_data_service_test.go

```
@@ -0,0 +1,76 @@
+	"github.com/kubescape/k8s-interface/k8sinterface"
+	"github.com/kubescape/k8s-interface/workloadinterface"
+	"github.com/kubescape/node-agent/pkg/objectcache"
+	"github.com/kubescape/node-agent/pkg/utils"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+)
+
+type fakeServiceClient struct {
+	namespace, name string
```

golangci-lint (2.12.2)

[error] field namespace is unused

(unused)